tests/test_cluster.go

```diff
@@ -162,6 +162,8 @@ func NewCluster(options *TestClusterConfig, logger log.Logger) (*TestCluster, er
 		esClient esclient.Client
 	)
 	if options.ESConfig != nil {
+		// Disable standard to elasticsearch dual visibility
+		pConfig.VisibilityStore = ""
```

Comment:
Could you elaborate on this change? Seems separate from the main change

Reply:
By default, this value is non-empty, which makes the internal visibility manager factory to build a dual visibility store. In my changes, when there's dual visibility, I return the index name of the first one.
Anyway, in the test, it sets up ES for visibility, so this setting is actually incorrect.